# Goal

geometry_msgs/Point target_point
---
# Feedback
float64 distance_to_target  # Current distance to the target

---
# Result
bool success       # True if the target was reached successfully, False otherwise
